Sync and add 'padding-inline' UA stylehseet rules for `mfrac` in mathml.css

Ahmad-S792 opened the pull request:
#### 612aa962c630f31a2e5feca8b0df7b4b344a9eab
<pre>
Sync and add &apos;padding-inline&apos; UA stylehseet rules for `mfrac` in mathml.css

<a href="https://bugs.webkit.org/show_bug.cgi?id=262789">https://bugs.webkit.org/show_bug.cgi?id=262789</a>
<a href="https://rdar.apple.com/problem/116922348">rdar://problem/116922348</a>

Reviewed by Frédéric Wang.

This patch aligns WebKit with Web-Specification [1] by adding &apos;padding-inline-*&apos; for `mfrac`.

[1] <a href="https://w3c.github.io/mathml-core/#user-agent-stylesheet">https://w3c.github.io/mathml-core/#user-agent-stylesheet</a>

* Source/WebCore/css/mathml.css:
* LayoutTests/platform/mac/mathml/presentation/roots-expected.txt: Rebaselined
* LayoutTests/platform/glib/mathml/presentation/roots-expected.txt: Rebaselined
* LayoutTests/platform/gtk/mathml/presentation/roots-expected.png: Rebaselined
* LayoutTests/platform/ios/mathml/presentation/roots-expected.png: Rebaselined
* LayoutTests/platform/ios/mathml/presentation/roots-expected.txt: Rebaselined
* LayoutTests/platform/mac-monterey/mathml/presentation/roots-expected.txt: Rebaselined
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/fractions/default-mfrac-padding-style.html: Add Test Case
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/fractions/default-mfrac-padding-style-expected.txt: Add Test Case Expectation

Canonical link: <a href="https://commits.webkit.org/271337@main">https://commits.webkit.org/271337@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d76eb5c5140e069829f182a55fa3b5916cc278f6

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/27970 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/6608 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/29271 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/30500 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/25514 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/28465 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/8602 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/3999 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/25278 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/28237 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/5361 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/24025 "Passed tests") | [❌ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/4618 "Found 1 new API test failure: /WPE/TestWebKitWebView:/webkit/WebKitWebView/notification (failure)") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/4800 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/25031 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/31189 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/25573 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/25479 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/31057 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/4807 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/2970 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/28869 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/6341 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/24798 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/6726 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/5252 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/5292 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on Source/WebCore/css/mathml.css:
```diff
@@ -155,6 +155,12 @@ ms, mtext, mi, mn, mo, annotation, mtd {
+/* Fractions */
+mfrac {
+    padding-inline-start: 1px;
+    padding-inline-end: 1px;
```
```suggestion
    padding-inline: 1px;
```

This can be shortened

Ahmad-S792 replied:
Done in 3f75cc7.

Ahmad-S792 committed:
Sync and add 'padding-inline' UA stylehseet rules for `mfrac` in mathml.css

https://bugs.webkit.org/show_bug.cgi?id=262789
rdar://problem/116922348

Reviewed by Frédéric Wang.

This patch aligns WebKit with Web-Specification [1] by adding 'padding-inline-*' for `mfrac`.

[1] https://w3c.github.io/mathml-core/#user-agent-stylesheet

* Source/WebCore/css/mathml.css:
* LayoutTests/platform/mac/mathml/presentation/roots-expected.txt: Rebaselined
* LayoutTests/platform/glib/mathml/presentation/roots-expected.txt: Rebaselined
* LayoutTests/platform/gtk/mathml/presentation/roots-expected.png: Rebaselined
* LayoutTests/platform/ios/mathml/presentation/roots-expected.png: Rebaselined
* LayoutTests/platform/ios/mathml/presentation/roots-expected.txt: Rebaselined
* LayoutTests/platform/mac-monterey/mathml/presentation/roots-expected.txt: Rebaselined
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/fractions/default-mfrac-padding-style.html: Add Test Case
* LayoutTests/imported/w3c/web-platform-tests/mathml/presentation-markup/fractions/default-mfrac-padding-style-expected.txt: Add Test Case Expectation

Canonical link: https://commits.webkit.org/271337@main
